core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1367,6 +1367,18 @@ public static String getSyncMetricName(long mountId) {
           .setDescription("The number of active write-RPCs managed by workers")
           .setMetricType(MetricType.COUNTER)
           .build();
+  public static final MetricKey CLUSTER_BLOCK_READER_THREAD_ACTIVE_COUNT =
```

Comment:
```suggestion
  public static final MetricKey CLUSTER_BLOCK_READER_ACTIVE_THREADS_COUNT =
```

Reply:
@beinan thanks for your review.

> I'm not a big fan of aggregating the metrics in master. As a long term plan, I prefer do the metrics aggregation on prometheus or something similar.

Yeah, that is a more general way. 
I opened another issue to track it #16752 , please comment there if you have other thoughts.